en/upgrade-tidb-operator.md

```diff
@@ -71,98 +82,104 @@ This document describes how to upgrade TiDB Operator and Kubernetes.
 
     > **Note:**
     >
-    > After TiDB Operator is upgraded, the `discovery` deployment in all TiDB clusters will be automatically upgraded to the corresponding version of TiDB Operator.
+    > After TiDB Operator is upgraded, the `discovery` Deployment in all TiDB clusters is automatically upgraded to the corresponding version of TiDB Operator.
```

Comment:
@DanielZhangQD Is it necessary to capitalize "Deployment"? If no special reasons, I would suggest keeping it as an ordinary word without capitalization.

Reply:
Confirmed. Just keep the capitalization.